src/ui/src/builder/journal/BuilderJournalHeader.vue

```diff
@@ -0,0 +1,73 @@
+<template>
+	<div class="BuilderJournalHeader">
+		<BuilderJournalSearchbar
+			@update:search="(val) => emit('update:search', val)"
+			@update:statuses="(val) => emit('update:statuses', val)"
+			@update:triggers="(val) => emit('update:triggers', val)"
```

Comment:
I think you want to use v-model here. It creates the emits logic for you


```suggestion
      v-model:search="search"
      v-model:statuses="statuses"
      v-model:triggers="triggers"
```

BTW, your current code is equivalent to


```suggestion
			@update:search="$emit('update:search', $event)"
			@update:statuses="$emit('update:statuses', $event)"
			@update:triggers="$emit('update:triggers', $event)"
```